src/w3iProxy/w3iPushFacade.ts

```diff
@@ -9,17 +9,15 @@ import type {
   W3iPush
 } from './pushProviders/types'
 import ExternalPushProvider from './pushProviders/externalPushProvider'
-import AndroidPushProvider from './pushProviders/androidPushProvider'
-import iOSPushProvider from './pushProviders/iosPushProvider'
 import InternalPushProvider from './pushProviders/internalPushProvider'
 import type { PushFacadeEvents } from './listenerTypes'
 
 class W3iPushFacade implements W3iPush {
   private readonly providerMap = {
     internal: InternalPushProvider,
     external: ExternalPushProvider,
-    android: AndroidPushProvider,
-    ios: iOSPushProvider
```

Comment:
Can we also delete the `iOSPushProvider`?

Reply:
There is no `iOSPushProvider`